src/Components/Aspire.OracleManagedDataAccess/OracleManagedDataAccessSettings.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.OracleManagedDataAccess;
+
+/// <summary>
+/// Provides the client configuration settings for connecting to a Oracle database using Oracle.ManagedDataAccess.
+/// </summary>
+public sealed class OracleManagedDataAccessSettings
+{
+    /// <summary>
+    /// The connection string of the Oracle database to connect to.
+    /// </summary>
+    public string? ConnectionString { get; set; }
+
+    /// <summary>
+    /// <para>Gets or sets a boolean value that indicates whether the database health check is enabled or not.</para>
+    /// <para>Enabled by default.</para>
+    /// </summary>
+    public bool HealthChecks { get; set; } = true;
+
+    /// <summary>
+    /// <para>Gets or sets a boolean value that indicates whether the Open Telemetry tracing is enabled or not.</para>
+    /// <para>Enabled by default.</para>
+    /// </summary>
+    public bool Tracing { get; set; } = true;
+
+    /// <summary>
+    /// <para>Gets or sets a boolean value that indicates whether the Open Telemetry metrics are enabled or not.</para>
+    /// <para>Enabled by default.</para>
+    /// </summary>
+    public bool Metrics { get; set; } = true;
```

Comment:
Are these used? If not, we shouldn't have properties for them.

Reply:
Removed for now, I looked in the ODP.NET documentation and there are still no metrics that we can use here.